cabal-testsuite/PackageTests/ExtraProgPath/pkg-config

```diff
@@ -0,0 +1,3 @@
+#/usr/bin/sh
```

Comment:
Note that this line is just a comment.  Maybe delete it.

As regards to fixing this towards a shebang: Can this be made portable so it also finds the Git Bash on Windows?